shared/templates/dconf_ini_file/template.py

```diff
@@ -1,6 +1,7 @@
 
 import os
 
+
```

Comment:
nit: Let's get rid of this extra space.

Reply:
@JAORMX When opening a PR previously, we recieved a linting error on this file: [pep8 E501 expected 2 blank lines](https://www.python.org/dev/peps/pep-0008/#blank-lines). 

Do you still want us to change this back?

Reply:
Nah, that's alright; let's keep it then